lib/msun/man/cospi.3

```diff
@@ -62,27 +62,21 @@ where p is the floating\(enpoint precision of
 .Ar x ,
 then the returned value is 1 and it has no significance.
 .Sh SPECIAL VALUES
-.Bl -tag
-.It
-.Fn cospi \*(Pm0
+.Bl -tag -width 11n
```

Comment:
I know that there's a preference for something like `-width "cospi blargle"`.  However, the item heads in this list contain rather special escaped roff commands, which don't make sense inside a `""`.  I therefore opted to go with the lowest `n` such that it displayed correctly in the terminal.

Optionally, we could make them all `indent-two` instead; that's 12 rather than 10 or 11 chars.

Reply:
I rewrote these to use `-width "cospi(+n/2)"`.

The suggestion to use rendered output for the `-width` argument is brilliant.

Reply:
I'm sorry I wasn't clear when I explained it the previous times, this is entirely my failure to communicate this with you. I didn't come up with this, this is how it's always been afaik. It's the width of the rendered longest item in the list.